### PR TITLE
Adds leather satchels and wallets to the HoP office all across the maps.

### PR DESCRIPTION
**Pull request by DATA-xPUNGED**
## About The Pull Request

This adds either a leather satchel (when there is space for it) or a simple wallet to every HoP's office.
## Why It's Good For The Game

HoP is the bureaucracy role that carries a bunch of tiny things. It should be helpful to have a wallet easily available
As for the leather satchel, HoP (and most of service, if i'm honest) Does not have a special departmental satchel/backpack, so they spawn in with a boring gray one. Having the option to quickly switch to a more stylish version is nice, instead of going to dorms to steal one from there. Note that the leather satchel is not on all maps, as some have an office too small to fit one, but i think small map variety like this is fine.
## Changelog
:cl:
qol: Leather satchels and wallets have been issued to the offices of Head of Personnel all across the sector. Studies suggest a fashion increase of 34%
/:cl:

### CODE REVIEW

**Comment by jughu** [comment]
isnt there a bunch of them at dorms and makeable woth leather?

**Comment by DATA-xPUNGED** [comment]
> isnt there a bunch of them at dorms and makeable woth leather?

There is (except on birdshot, that one's dorms fucking suck) But heads should have the privilege to not need to walk all the way to dorms and/or wait for the biogen to get produce.

**Comment by jughu** [comment]
(wallet is just a buff for hops to slot 4 head ID's in to increase their access region) besides that this is a good pr

**Comment by EricZilla** [comment]
Agreed on wallet stealth buff but also as a hop I constantly have to get people new bags when they suffer a total loss, so having a rack of spare satchels is pretty nice

File: _maps/map_files/IceBoxStation/IceBoxStation.dmm [hunk 1]
```diff
@@ -69574,6 +69577,13 @@
 	},
 /turf/open/floor/iron/white,
 /area/station/science/robotics/lab)
+"wjJ" = (
+/obj/item/pen{
+	pixel_y = 4;
+	pixel_x = 5
+	},
+/turf/open/genturf,
+/area/icemoon/underground/unexplored/rivers/deep)
```
Comment (Maurukas) [inline _maps/map_files/IceBoxStation/IceBoxStation.dmm:69586]: This is at approximately (101, 114)  (105, 118) on the bottom z of Icebox. It is a random pen that has somehow managed to creep into a genturf area

**Review by Maurukas — CHANGES_REQUESTED** [review]
Random cursed mapcruft on the bottom layer of Icebox, otherwise looks good.

File: _maps/map_files/IceBoxStation/IceBoxStation.dmm (hunk 1 above)
Comment (DATA-xPUNGED) [reply _maps/map_files/IceBoxStation/IceBoxStation.dmm:69586]: I used Find On Map and i still can't find it, what the fuck

Comment (DATA-xPUNGED) [reply _maps/map_files/IceBoxStation/IceBoxStation.dmm]: I've manually edited it in code as i cannot for the life of me find it on strongDMM, i HOPE with works

**Review by Maurukas — APPROVED** [review]

**Pull request merged**